src/action-menu/ActionMenuGroup.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './ActionMenu.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  children: PropTypes.node,
```

Comment:
Is children custom? Should we call out these component should expect onArrow, onChar and indentChildren props?